src/Tools/ExternalAccess/OmniSharp/GoToDefinition/OmniSharpFindDefinitionService.cs

```diff
@@ -15,7 +15,10 @@ internal static class OmniSharpFindDefinitionService
 {
     internal static async Task<ImmutableArray<OmniSharpNavigableItem>> FindDefinitionsAsync(Document document, int position, CancellationToken cancellationToken)
     {
-        var service = document.GetRequiredLanguageService<INavigableItemsService>();
+        var service = document.GetLanguageService<INavigableItemsService>();
+        if (service is null)
+            return ImmutableArray<OmniSharpNavigableItem>.Empty;
```

Comment:
just being paranoid.  this is omnisharp.  so presumably only for C#/VB.  But i'm not taking chances.

Reply:
Note even VB, just C#. But I'm ok with paranoia.